src/schema/v2/searchCriteriaLabel.ts

```diff
@@ -174,11 +183,37 @@ function getSizeLabels(sizes: string[]) {
 
   return sizes.map((size) => ({
     name: "Size",
-    value: toTitleCase(size.toLowerCase()), // TODO: this a placeholder, we need to format these properly
+    value: SIZES[`${size}`],
     field: "sizes",
   }))
 }
 
+const convertToCentimeters = (element: number) => {
+  return Math.round(element * ONE_IN_TO_CM)
```

Comment:
Q: do we use the same rounding logic in other parts of the system?

Reply:
Good question - yeah, we do. For example, if you visit [this Force page](https://www.artsy.net/artist/zhao-zhao/works-for-sale?width=3.937007874015748-39.37007874015748) where a custom size filter has been applied, you can see that the URL has un-rounded values, while the label above the grid has rounded them off to the nearest centimeter.

This function is copied basically verbatim from [Force's SizeFilter file](https://github.com/artsy/force/blob/d85e7e8bc7928489c707000a92e66d726450a194/src/v2/Components/ArtworkFilter/ArtworkFilters/SizeFilter.tsx#L58)